tests/schema/test_baseschema.py

```
@@ -4123,3 +4123,290 @@ def test_from_dict_manifest_clear_multiple_calls():
     assert schema.get("data", "key1", "value") == "first"
     # key3 is cleared by second from_dict
     assert schema.get("data", "key3", "value") is None
+
+
+def test_from_dict_with_failure():
+    class DummySchema1(BaseSchema):
+        def __init__(self, requiredkey):
```

Ruff (0.15.2)

[warning] Unused method argument: `requiredkey`

(ARG002)